src/Theme.cpp

```diff
@@ -215,3 +215,14 @@ QStringList Theme::userThemes() {
 	QDir directory(themeDirectory());
 	return directory.entryList(QStringList() << "*.ini", QDir::Files);
 }
+
+QString Theme::themeThame(const QString &theme_file) {
```

Comment:
River style, directly [from London](https://en.wikipedia.org/wiki/Thames) ;) I guess you meant `themeName()`.

Reply:
LOL, indeed!

Reply:
All fixed in latest push :-)